test.js

```diff
@@ -1230,3 +1231,70 @@ describe('concurrency', function() {
     expect(ParallelApi.jobs).to.equal(17);
   });
 });
+
+describe('getBabelVersion()', function() {
+  it ('returns the correct version', function() {
+    var babelCorePath = path.join(__dirname, 'node_modules', 'babel-core', 'index.js');
```

Comment:
I believe the following would be simpler:

```
var version = require(__dirname + 'node_modules/babel-core/package.json').version
```

Reply:
yeah much simpler than what I had